tests/integration/defs/accuracy/test_step3p7_gsm8k_canary.py

```
@@ -0,0 +1,477 @@
+        info["exception_message"] = str(e)[:1024]
+        info["traceback_tail"] = traceback.format_exc().splitlines()[-12:]
+        info["elapsed_s"] = round(time.time() - t0, 1)
+        return 3, info
+    info["instantiation_s"] = round(time.time() - t0, 1)
+    try:
+        sampling = SamplingParams(temperature=0.0, top_k=1, max_tokens=args.max_new_tokens)
+        prompts = [p for p, _ in probes]
+        outputs = llm.generate(prompts, sampling_params=sampling)
+        per_prompt = []
+        correct = 0
+        for (prompt, expected), out in zip(probes, outputs):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)